app/src/main/java/ru/qveex/rst_tur/presentation/screens/blog/BlogScreen.kt

```diff
@@ -96,30 +98,30 @@ fun BlogScreen(
                         .clip(RoundedCornerShape(size = 8.dp)),
                     bitmap = bitmap,
                     contentScale = ContentScale.Crop,
-                    contentDescription = "Blog fun photo"
+                    contentDescription = stringResource(R.string.blog_fun_photo_content_description)
                 )
                 Text(
-                    text = blogViewModel.blog?.date?.dateConvert() ?: "",
+                    text = blogViewModel.blog?.date?.dateConvert() ?: stringResource(R.string.emptyString),
```

Comment:
```suggestion
                    text = blogViewModel.blog?.date?.dateConvert() ?: "",
```

I think string res in these cases is redundant, because an empty string will always be empty  ¯\_(ツ)_/¯
And `""` can use string pull and don't waste time and memory to getting android resources